front50-cassandra/front50-cassandra.gradle

```diff
@@ -21,6 +21,7 @@ configurations.all {
 dependencies {
   compile project(":front50-core")
   compile spinnaker.dependency('korkCassandra')
+  compile spinnaker.dependency("korkSwagger")
```

Comment:
Can we kill off this whole cassandra module yet, @ewiseblatt @robzienert ?

Reply:
I'm in favor of getting rid of cassandra.